src/test/crimson/test_socket.cc

```diff
@@ -41,15 +41,15 @@ class SocketFactoryBase
       lo.reuse_address = true;
       factory.listener = seastar::listen(s_addr, lo);
     }).then([this] {
-      return this->container().invoke_on_all([] (auto& factory) {
+      this->container().invoke_on_all([] (auto& factory) {
```

Comment:
Why this future is not chained any more (the `return` is removed) ?

Reply:
sorry, i was trying to address the test failure using the wrong way. this change is reverted in the latest revision.